infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -594,4 +572,31 @@ public static void cleanShutdown(@Nonnull NettyServerRouter router) {
         CompletableFuture.allOf(shutdownFutures).join();
         log.info("CleanShutdown: Shutdown Complete.");
     }
+
+    /**
+     * Print the corfu logo.
+     */
+    private static void printLogo() {
+        println(ansi().fg(WHITE).toString());
+        println("▄████████  ▄██████▄     ▄████████    ▄████████ ███    █▄");
+        println("███    ███ ███    ███   ███    ███   ███    ██████    ███");
+        println("███    █▀  ███    ███   ███    ███   ███    █▀ ███    ███");
+        println("███        ███    ███  ▄███▄▄▄▄██▀  ▄███▄▄▄    ███    ███");
+        println("███        ███    ███ ▀▀███▀▀▀▀▀   ▀▀███▀▀▀    ███    ███");
+        println("███    █▄  ███    ███ ▀███████████   ███       ███    ███");
+        println("███    ███ ███    ███   ███    ███   ███       ███    ███");
+        println("████████▀   ▀██████▀    ███    ███   ███       ████████▀ ");
+        println("                        ███    ███");
+        println(ansi().reset().toString());
+    }
+
+    /** Print an object to the console, followed by a newline.
+     *  Call this method instead of calling System.out.println().
+     *
+     * @param line  The object to print.
+     */
+    @SuppressWarnings("checkstyle:printLine")
+    private static void println(Object line) {
+        System.out.println(line);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)